applications/FluidDynamicsApplication/custom_processes/mass_conservation_check_process.cpp

```diff
@@ -77,6 +77,10 @@ std::string MassConservationCheckProcess::Initialize(){
     double inter_area = 0.0;
     const auto& r_comm = mrModelPart.GetCommunicator().GetDataCommunicator();
 
+    ModelPart& non_const_model_part = const_cast<ModelPart&>(mrModelPart);
+    auto nodal_h_process = FindNodalHProcess<false>(non_const_model_part);
+    nodal_h_process.Execute();
+
```

Comment:
I think this is a leftover from one of our preliminary experiments.

Reply:
You are right. Done!